modules/API/external/external_api.js

```diff
@@ -200,6 +200,8 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * authentication.
      * @param {string} [options.onload] - The onload function that will listen
      * for iframe onload event.
+     * @param {Array<Object>} [options.invites] - Array of objects containing
```

Comment:
Can we not have this here? It's already polluted enough as it is... and the API user can trigger this themselves at any point really.